json-schema/json-schema/src/main/scala/endpoints/algebra/Codec.scala

```diff
@@ -68,4 +67,34 @@ object Codec {
       Decoder.sequentially(ab)(bc)
     )
 
+  /** Produce a codec to/from a string. If the parsing function fails, the
+    * decoding output is an [[Invalid]] with a message mentioning the type name.
+    *
+    * @param type name of the type being decoded
+    * @param parse parsing function to use, with exceptions turned into [[Invalid]]
+    * @param print printing function to use, not supposed to throw exceptions
+    */
+  def parseStringCatching[A](
```

Comment:
Sorry to be nitpicking, but what about `parseStringCatchingExceptions` instead?

Reply:
That's actually a good name 👍. Done.